packages/pwa-kit-dev/src/configs/jest/jest.config.js

```diff
@@ -10,12 +10,13 @@ module.exports = {
     testURL: 'http://localhost/',
     verbose: true,
     collectCoverage: true,
+    coveragePathIgnorePatterns: ['app/application-extensions', '/node_modules/'],
```

Comment:
For projects that are generated with extracted application extensions we want to ignore the node modules in their locally installed package folder. This ignores everything, but [here](https://gus.lightning.force.com/a07EE000026NJhAYAW) is a ticket to correct this.